### ISSUE TITLE
Is the project maintained or discontinued? 

### ISSUE THREAD

**Issue by bork91** [aerogear/keycloak-metrics-spi#140]
Hi,
wondering if the project is maintained or it has been discontinued.

i see old release and old open issues.

**Comment by pb82** [comment]
Hey @bork91 
this project has fallen behind on maintenance but since native Keycloak metrics are not on the horizon yet, it is still valuable.

I'd like to set up some sort of bi-weekly maintenancy scheme / triage call to discuss issues, pull requests and releases.

Please let me know what you think. This would be open to anyone interested.

**Comment by vilmosnagy** [comment]
@pb82 i'm interested :)

**Comment by Tristan971** [comment]
Hi @pb82 I'd definitely be interested

**Comment by danielloader** [comment]
Yeah I've found the native metrics by design seem to skip all the "business logic" type metrics, logins, logouts, failures to login etc. It's all keycloak core metrics rather than the day to day goings on from a user side.
